IdentityCore/src/cache/token/MSIDAccountCacheItem.h

```diff
@@ -32,6 +32,7 @@
 @property (readwrite, nonnull) NSString *homeAccountId;
 @property (readwrite, nonnull) NSString *environment;
 @property (readwrite, nullable) NSString *localAccountId;
+@property (readwrite, nullable) NSString *tenantId;
```

Comment:
I think it's actually a bug on our side that realm is not tenantId already. 
I'd rather fix the bug and add a separate field for tenant alias if necessary.